src/content/posts/how-to-raise-funds/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: How to Raise Funds Using Angor
+published: 2024-06-17
+description: 'Angor is a decentralized crowdfunding platform on Bitcoin, offering secure fund management and direct investor engagement for transparent project funding.'
+image: './raise.svg'
+tags:  [Bitcoin, Technology, Education]
+category: 'Angor'
+draft: false 
+---
+### Raise Funds Securely with Angor:  Step-by-Step Guide
+
+Angor is a decentralized crowdfunding platform built on Bitcoin and uses Nostr for enhanced security and transparency. It allows founders to raise funds for their projects while maintaining control over the funds and fostering direct communication with investors. Here’s a step-by-step guide on how to raise funds using Angor.
+
+### Create a Wallet
+
+Before you can start raising funds, you need to set up a digital wallet on Angor. If you don’t have a wallet yet, create a wallet to use Angor. Follow the on-screen instructions to create your wallet.
+- Navigate to the wallet creation section.
+- Click on "Create Wallet."
+- Angor will automatically set up the wallet for you.
+- Set a strong password to protect your wallet.
+
+  ![Wallet after creation](./assets/wallet-after-creation.svg)
```

Comment:
I think we also need an image here for the wallet popup (before creation) otherwise it is a bit confusing to the user they see a popup and don't know if it is correctly part of the flow

Reply:
It would be more clear then